Doc/whatsnew/3.9.rst

```diff
@@ -616,6 +616,10 @@ Deprecated
   `parso`_.
   (Contributed by Carl Meyer in :issue:`40360`.)
 
+* The :mod:`xml.etree.cElementTree` module has been deprecated since Python
+  3.3 and is scheduled to be removed in Python 3.10.
+  (Contributed by Serhiy Storchaka and Christian Heemes in :issue:`36543`)
```

Comment:
A typo in your own name? That's rare, I guess. ;-)

Reply:
You wouldn't believe me :+1: